Migrations/ApplicationDbContextModelSnapshot.cs

```diff
@@ -193,6 +193,12 @@ protected override void BuildModel(ModelBuilder modelBuilder)
 
                     b.Property<DateTime>("TimeSent");
 
+                    b.Property<string>("TimeSentNative");
```

Comment:
This should probably be regenerated - these properties (`TimeSentNative`, `ThreadID`, `ThreadIDNative`) are out of date.